src/components/command-bar/commands/search/unified-search/components/unified-hits-container/index.tsx

```diff
@@ -14,58 +11,36 @@ import {
 } from '../../../components'
 // Unified search
 import { UnifiedHit } from '../unified-hit'
-import { gatherSearchTabsContent } from './helpers'
 // Types
-import type { ProductSlug } from 'types/products'
 import type { Hit } from 'instantsearch.js'
 import type { UnifiedSearchTabContent } from './helpers'
 // Styles
 import s from './unified-hits-container.module.css'
 
 /**
  * Render search results from our unified index into content-type tabs.
- *
- * Note: this component needs to be used within an `InstantSearch` container
- * imported from 'react-instantsearch-hooks-web'. That container provides
- * the context from which `rawHits` are pulled.
  */
 export function UnifiedHitsContainer({
-	currentProductSlug,
+	tabsData,
 	suggestedPages,
 }: {
-	currentProductSlug?: ProductSlug
+	tabsData: UnifiedSearchTabContent[]
 	suggestedPages: SuggestedPage[]
 }) {
-	const { hits: rawHits } = useHits()
-
-	/**
-	 * Transform searchableContentTypes into data for each content tab.
-	 *
-	 * Note: we set up this data before rather than during render,
-	 * because each tab needs data from all other tabs in order
-	 * to show a helpful "No Results" message.
-	 */
-	const allTabData = useMemo(
-		() => gatherSearchTabsContent(rawHits, currentProductSlug),
-		[rawHits, currentProductSlug]
-	)
-
-	/**
-	 * Render the tabs. This is mostly presentation since `tabData` logic is done.
-	 */
 	return (
 		<div className={s.tabsWrapper}>
 			<Tabs showAnchorLine={false} variant="compact">
-				{allTabData.map((tabData: UnifiedSearchTabContent) => {
-					const { type, heading, icon, hits, hitCount, otherTabsWithResults } =
-						tabData
+				{tabsData.map((tabData: UnifiedSearchTabContent) => {
+					const { type, heading, icon, hits, hitCount, otherTabData } = tabData
 					const resultsLabelId = `${type}-search-results-label`
-
 					return (
 						<Tab
 							heading={heading}
 							headingSlot={
-								<TabHeadingWithCount heading={heading} count={hitCount} />
+								<TabHeadingWithCount
+									heading={heading}
+									count={type === 'global' ? undefined : hitCount}
+								/>
```

Comment:
We hide the tab count for the `global` tab - it doesn't "add up" with the other tabs, so felt more confusing than helpful to include it.